packages/thriver-core-components/lib/component-tabs/tabs.js

```diff
@@ -16,16 +16,16 @@ var toggleTabs = function (event) {
 	// Remove active state from all links
 	for (let i = 0; i < links.length; ++i)
 		Thriver.util.makeActive(links[i], false);
-	
+
 	// Hide all sections
 	for (let i = 0; i < sections.length; ++i)
 		Thriver.util.hide(sections[i], true);
-        
+
     // Set link as active
     Thriver.util.makeActive(event.target, true);
 
     // Set section as active
-    Thriver.util.hide( menu.parentElement.querySelector('article[data-id="' + 
+    Thriver.util.hide( menu.parentElement.querySelector('article[data-id="' +
```

Comment:
We should use ECMAScript 6 templates now :-D

    Thriver.util.hide(menu.parentElement.querySelector(`article[data-id="${event.target.dataset.id}"]`), false);